<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lecteurvue</class>
 <widget class="QMainWindow" name="lecteurvue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lecteurvue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="l_categorie">
        <property name="text">
         <string>Catégorie</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="l_rang">
        <property name="text">
         <string>Rang</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_titreImage">
        <property name="text">
         <string>Titre Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="l_titreDiapo">
        <property name="text">
         <string>Titre Diapo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="b_Reculer">
        <property name="text">
         <string>Reculer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QOpenGLWidget" name="openGLWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="b_Avancer">
        <property name="text">
         <string>Avancer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_ActiverModeAuto">
        <property name="text">
         <string>Mode Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_ActiverModeManuel">
        <property name="text">
         <string>Mode Manuel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_ActiverBoucle">
        <property name="text">
         <string>Boucle</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <addaction name="actionChanger_Diaporama"/>
    <addaction name="actionVitesse_de_d_filement"/>
    <addaction name="actionEnlever_Diaporama"/>
   </widget>
   <widget class="QMenu" name="menuA_propos_de">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParam_tres"/>
   <addaction name="menuA_propos_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChanger_Diaporama">
   <property name="text">
    <string>Changer Diaporama</string>
   </property>
  </action>
  <action name="actionVitesse_de_d_filement">
   <property name="text">
    <string>Vitesse de défilement</string>
   </property>
  </action>
  <action name="actionEnlever_Diaporama">
   <property name="text">
    <string>Enlever Diaporama</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos_de">
   <property name="text">
    <string>A propos de ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
